<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QWidget" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="backgroundFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>768</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1024</width>
      <height>768</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#backgroundFrame
{
background-image: url(:/pic/res/login_background.png);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="titleLabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>110</y>
       <width>521</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文新魏</family>
       <pointsize>35</pointsize>
      </font>
     </property>
     <property name="text">
      <string>末端试水装置管理端登录</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
    <widget class="QLabel" name="idLabel">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>270</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans CJK SC Regular</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>工号</string>
     </property>
    </widget>
    <widget class="QLabel" name="passwordLabel">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>350</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans CJK SC Regular</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginPushButton">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>470</y>
       <width>85</width>
       <height>38</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>11</red>
           <green>118</green>
           <blue>100</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>11</red>
           <green>118</green>
           <blue>100</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>11</red>
           <green>118</green>
           <blue>100</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ToolTipText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>11</red>
           <green>118</green>
           <blue>100</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>11</red>
           <green>118</green>
           <blue>100</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>11</red>
           <green>118</green>
           <blue>100</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ToolTipText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>11</red>
           <green>118</green>
           <blue>100</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>11</red>
           <green>118</green>
           <blue>100</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>11</red>
           <green>118</green>
           <blue>100</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ToolTipText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans CJK SC Regular</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(11, 118, 100);</string>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="idLineEdit">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>300</y>
       <width>300</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans CJK SC Regular</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordLineEdit">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>380</y>
       <width>300</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans CJK SC Regular</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="D:/managerclient/ManagerClient/res.qrc"/>
 </resources>
 <connections/>
</ui>
